jekyll/_cci2/ios-codesigning.md

```diff
@@ -106,6 +106,14 @@ Keys -> Add user key** and click *Authorize with GitHub*.
 same GitHub permissions as the user who will be clicking the *Authorize
 with GitHub* button.
 
+**Note2:** In your Matchfile, you must use **git** URL unlike **https://** for git_url parameter. 
+Otherwise you may see an authentication error when you attempt to use match.
```

Comment:
Let's change this to

> In your `Matchfile`, the `git_url` should be an **SSH** URL ( in the `git@github.com:...` format), rather than a **HTTPS** URL. Otherwise you may see authentication errors when you attempt to use match.